techmatters/flex-plugins
# CHI-2778 add flex UI support telegram channel

## Description

- New feature: This task involved:
-  Adding an Icon, updating the queue status, and setting up the Telegram channel on Flex UI.
- Removing the Twitter-related stuff from Flex UI.

### Checklist

### Other Related Issues

None

[CHI-2778](https://tech-matters.atlassian.net/browse/CHI-2778)

### Verification steps


- Go to the Telegram app and look for "Aselo Development"
- Start a conversation
- When the task comes in on Aselo, verify that it's reflected under the `TG` task queue.

### AFTER YOU MERGE

1. Cut a release tag using the Github workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P

[CHI-2778]: https://tech-matters.atlassian.net/browse/CHI-2778?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
